docs/sdk_python/create_model.mdx

```diff
@@ -25,17 +25,32 @@ my_model = project.create_model (
 )
 ```
 
-Here is the syntax to create an OpenAI model:
+Please note that in the case of LLM models, the parameters are stored in `options`. Here is the syntax to create an OpenAI model:
 
 ```bash
-sentiment_classifier = project.create_model(
+sentiment_classifier = project.create_model (
       name='sentiment_classifier',
-      engine='huggingface',
+      engine='openai', # alternatively: engine=server.ml_engines.openai
       predict='sentiment',
+      prompt_template='answer this question: {{questions}}'
       options={
-          'model_name':'cardiffnlp/twitter-roberta-base-sentiment',
-          'input_column': 'comment',
-          'labels': ['negative', 'neutral', 'positive']
+          'model_name':'gpt4'
```

Comment:
it is possible to move 'model_name' also from options. All unknown params are recognized as options now. So these commands are equal:

```
sentiment_classifier = project.create_model (
      name='sentiment_classifier',
      engine='openai', # alternatively: engine=server.ml_engines.openai
      predict='sentiment',
      options={
          'prompt_template':'answer this question: {{questions}}',
          'model_name':'gpt4'
      }
)

sentiment_classifier = project.create_model (
      name='sentiment_classifier',
      engine='openai', # alternatively: engine=server.ml_engines.openai
      predict='sentiment',
      prompt_template='answer this question: {{questions}}',
      model_name = 'gpt4'
)
```